packages/ant-component-mapper/src/files/time-picker.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TimePicker as AntTimePicker } from 'antd';
+import { validationError } from '../common/helpers';
+import { meta, input } from '@data-driven-forms/common/src/prop-types-templates';
+import { useFieldApi } from '@data-driven-forms/react-form-renderer';
+import AntForm from '../common/form-wrapper';
+
+const DatePicker = (props) => {
+  const { input, isReadOnly, isDisabled, placeholder, isRequired, label, helperText, description, validateOnMount, meta } = useFieldApi(props);
+  const invalid = validationError(meta, validateOnMount);
+  const placeholderDisplay = placeholder ? placeholder : 'Select time';
+  const help = invalid || helperText || description;
+  return (
+    <AntForm layout="vertical" isRequired={isRequired} label={label} invalid={invalid} help={help}>
+      <AntTimePicker
+        use12Hours
```

Comment:
`format` and `use12Hours` should be passed via props (you can set their default values)